event_manager/settings.py

```diff
@@ -54,6 +55,9 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+
+    # TODO Disabled this csrf token for now, we will going to enable it afterwards
```

Comment:
@usman-hussain please create a ticket for this

Reply:
@mashhoodr Here is the ticket
https://github.com/recurship/event-manager/issues/39